hydroflow_datalog_core/src/lib.rs

```diff
@@ -700,7 +709,7 @@ fn apply_aggregations(
                 TargetExpr::Aggregation(Aggregation::CountUnique(..)) => {
                     let idx = syn::Index::from(agg_idx);
                     after_group_lookups
-                        .push(parse_quote_spanned!(get_span(field.span)=> a.#idx.unwrap().len()));
+                        .push(parse_quote_spanned!(get_span(field.span)=> a.#idx.unwrap().1));
```

Comment:
You might also just be able to do `x.len() as _`

Reply:
Unfortunately, we get type inference complaints that way, because we want this type to default to `i32` if there are no other constraints.